test/functional/p2p_tx_download.py

```diff
@@ -42,18 +46,13 @@ def on_getdata(self, message):
                 self.tx_getdata_count += 1
 
 
-# Constants from net_processing
-GETDATA_TX_INTERVAL = 60  # seconds
-INBOUND_PEER_TX_DELAY = 2  # seconds
-TXID_RELAY_DELAY = 2 # seconds
-OVERLOADED_PEER_DELAY = 2 # seconds
-MAX_GETDATA_IN_FLIGHT = 100
+# Constants from txdownloadman
+MAX_PEER_TX_REQUEST_IN_FLIGHT = 100
 MAX_PEER_TX_ANNOUNCEMENTS = 5000
```

Comment:
nit: Not a blocker, but it seems like it would make sense for all of these types of constants to live in `p2p.py`.

Reply:
I don't have a strong opinion on this. 

I've checked and looks like no other test uses this, or re-defines it. I'd lean towards leaving it locally to avoid bloating `p2p.py`, but I'm open to move it if there is support for it